package.json

```diff
@@ -30,12 +30,13 @@
     "scripts": {
         "start": "cross-env PORT=3001 react-scripts start",
         "build": "react-scripts build",
-        "test": "react-scripts test",
-        "test-once": "cross-env CI=true react-scripts test",
+        "test": "react-scripts test --coverage",
```

Comment:
Let's not add the `--coverage` flag here (unless it's needed for some reason, but `test-once` should be enough for the CI). It will make the tests slower.

Reply:
it's done